blog/forge-v6-release.md

```diff
@@ -0,0 +1,189 @@
+
+Visit the [Why Electron Forge] section of our docs to learn more about Forge and the benefits that it offers.
+
+## What does this release mean for Forge?
+
+### A complete rewrite
+
+From v1 to v5, Electron Forge was based on the now-discontinued [`electron-compile`](https://www.npmjs.com/package/electron-compile) project. Forge v6 is a complete rewrite of the project with a new modular architecture that can be extended to meet any Electron application's needs. In the past few years, Forge v6.0.0-beta has achieved feature parity with v5 and code churn has slowed down dramatically, making the tool ready for general adoption.
+
+### Making it official
+
+Historically, the Electron project has been unopinionated about build tooling, leaving the task to various community-maintained packages. However, with Electron maturing as a project, it has become harder for new Electron developers to understand which tools they need to build and distribute their apps to users.
+To help guide Electron developers in the distribution process, **we have have decided to make Forge the official batteries-included build pipeline for Electron**. Over the past year, we have been slowly integrating Forge into the official Electron documentation, and we have most recently moved Forge over from its old home in electron-userland/electron-forge to the [electron/forge](https://github.com/electron/forge) repo.
+
+## Installing / upgrading to Forge v6
+
+:::info
+
+This section assumes you have a pre-existing project. If you are looking to get started with a new Forge project, head on to the [Getting Started] section in the docs. 
+
+:::
+
+Forge will try to migrate from Electron Forge v5 automatically as much as possible using the same process as importing an existing Electron app to Forge. Some of it may need to be migrated manually. Importing your settings can be done using the [CLI API] node module, which at a high level is a wrapper for the Forge [Core API] that is accessible from the terminal.
+
+<Tabs>
+  <TabItem value="Yarn" label="Yarn" default>
+
+```
+   cd my-app
+   yarn add --dev @electron-forge/cli
+   yarn electron-forge import
+```
+
+  </TabItem>
+  <TabItem value="NPM" label="NPM">
+
+```
+   cd my-app
+   npm install --save-dev @electron-forge/cli
+   npm exec --package=@electron-forge/cli -c "electron-forge import"
+```
+
+  </TabItem>
+</Tabs>
+
+Manual migration details can be found in the Forge [import documentation]. Further help can be offered in our [Discord](https://discord.gg/f4cH9BzaDw).
+
+## Why switch to Forge from an existing pipeline?
+
+If your app already uses an existing build tooling solution that provides packaging and publishing capabilities, the benefits associated with adopting Electron Forge can still outweigh the initial switching cost.
```

Comment:
I'd be more general in the intro sentence

```suggestion
If you already have tooling for packaging and publishing your Electron app, the benefits associated with adopting Electron Forge can still outweigh the initial setup cost.
```

Reply:
Gonna keep the word `switching` cost, I think it first the definition of what the user is doing a bit more (nit)
 feel free to push back if you feel strongly abt it though